config/config.exs

```diff
@@ -31,7 +31,7 @@ use Mix.Config
 
 config :sdn_epc,
   channel_id: "1",
-  controller_ip: "192.168.56.101",
+  controller_ip: '192.168.56.101',
```

Comment:
Ahh if you're reading the IP from the config file it's better to have it as a tuple from the begging.